src/main.js

```diff
@@ -45,7 +47,7 @@ csgo.on("connectedToGC", () => {
 client.on("receivedFromGC", (appid, messageType) => {
   logger.info(`Received: ${appid}, ID: ${messageType}`);
   if (messageType == BETA_ENROLL_MESSAGE_TYPE) {
-    logger.warn(`${userName} RECEIVED BETA INVITE!`);
+    logger.warn(`RECEIVED BETA INVITE!`);
```

Comment:
userName was never really defined. 
If an user env is given, it will be present in the meta.